swagger.json

```diff
@@ -902,6 +906,27 @@
         "operationId" : "getOrgs",
         "consumes" : [ "application/json" ],
         "produces" : [ "application/json" ],
+        "parameters" : [ {
+          "name" : "filter[manager]",
+          "in" : "query",
+          "description" : "Identifiers for users which will have the organizations they manage represented in the response",
```

Comment:
This seems like a very roundabout way of describing this - what I really want to know as an API consumer is "Only show organizations that are managed by this user"